.github/workflows/inspektor-gadget.yml

```diff
@@ -137,6 +137,10 @@ jobs:
       run: |
         make test
 
+    - name: Unit tests requiring root
+      run: |
+        ENABLE_GADGET_TRACER_MANAGER_TESTS=true sudo -E go test ./pkg/gadgettracermanager/
```

Comment:
The GitHub action fails with:
```
compile: version "go1.16.6" does not match go tool version "go1.15.14"
FAIL
```

I should find a way to make it a regular unit test that does not require root...

Reply:
I took a different approach for tests at https://github.com/kinvolk/inspektor-gadget/pull/213